src/server/server.js

```diff
@@ -23,6 +24,8 @@ app.use(webpackDevMiddleware(compiler, {
 app.get('/', (req, res) => res.send('Hello World!'));
 
 app.get('/api/tasks', (req, res) => new TaskManager(req, res).find());
-app.get('/api/tasks/:id', (req, res) => new TaskManager(req, res).findById(req.params.id));
+app.get('/api/task/:id', (req, res) => new TaskManager(req, res).findById(req.params.id));
+app.delete('/api/task/:id', (req, res) => new TaskManager(req, res).deleteTask(req.params.id));
+app.put('/api/task/:id', (req, res) => new TaskManager(req, res).updateRecords());
```

Comment:
Why is this here? As far as I can tell, there's no `updateRecords` function in the task manager.

Reply:
sure, lemi reslove it

Reply:
sure, lemi resolve it